src/sentry/seer/similarity/utils.py

```diff
@@ -17,12 +17,23 @@
 FULLY_MINIFIED_STACKTRACE_MAX_FRAME_COUNT = 20
 SEER_ELIGIBLE_PLATFORMS_EVENTS = frozenset(
     [
-        "csharp",
```

Comment:
@lobsterkatie pointed me to this short list:
https://github.com/getsentry/relay/blob/20f72be4c573355ea0374bcaf5a3ff3fbabe29d5/relay-event-schema/src/protocol/constants.rs#L2C11-L22